src/BusinessLayer/SongControl/SongControl.cpp

```diff
@@ -1,13 +1,14 @@
 #include <QDir>
-#include "SongControl.h"
+#include <dirent.h>
```

Comment:
Are we able to drop this now?

Reply:
<dirent.h>? I didn't remove the usage of dirent so we can't remove it yet. I'll leave that as a future bug/refactor.

Reply:
Make a new issue for it!